nb_anacondacloud/static/main.js

```diff
@@ -25,7 +26,7 @@ define(['jquery', 'base/js/dialog'], function ($, dialog) {
         }
 
         $.ajax({
-            url: "/ac-publish",
+            url: Jupyter.notebook.base_url + "ac-publish",
```

Comment:
this is important and, in fact, impacted in other extensions: https://github.com/Anaconda-Server/nb_revisioncontrol/pull/5/files#diff-44ce3f32fb6ea0ec59591a50d801ff67R43